tensorflow/core/platform/cloud/retrying_file_system.cc

```diff
@@ -96,7 +102,7 @@ Status RetryingFileSystem::NewRandomAccessFile(
   TF_RETURN_IF_ERROR(CallWithRetries(std::bind(&FileSystem::NewRandomAccessFile,
                                                base_file_system_.get(),
                                                filename, &base_file)));
-  result->reset(new RetryingRandomAccessFile(std::move(base_file)));
+  result->reset(new RetryingRandomAccessFile(std::move(base_file), 0));
```

Comment:
My understanding was that you wanted to pass zero for the initial_delay so that the unit tests could run faster?